src/tool_writeout.c

```diff
@@ -275,8 +282,8 @@ void ourWriteOut(CURL *curl, struct OutStruct *outs, const char *writeinfo)
                   fprintf(stream, "%ld", longinfo);
                 break;
               case VAR_EFFECTIVE_FILENAME:
-                if(outs->filename)
-                  fprintf(stream, "%s", outs->filename);
+                if(per->outs.filename)
+                  fprintf(stream, "%s", per->outs.filename);
```

Comment:
You can also use fputs. I don't think it's necessary to have the printf overhead here

Reply:
You are right, but this is a thing of the past and not changes I made, but I will change it.

Reply:
Sorry but I don't know what you mean

Reply:
I mean I didn't wrote `fprintf(stream, "%s", `.

Reply:
Still something you can change :-)